client/src/game/ui/arrows.ts

```diff
@@ -235,9 +236,13 @@ const animate = (arrow: Arrow, card: HanabiCard, rot: number, giver: number, tur
   }).play();
 };
 
-export const click = (event: any, order: ReplayArrowOrder, element: any) => {
+export const click = (
+  event: KonvaEventObject<MouseEvent>,
+  order: ReplayArrowOrder,
+  element: any,
+) => {
   if (
-    event.evt.which === 3 // Right-click
+    event.evt.button === 3 // Right-click
```

Comment:
does this still work the same as button?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which

The MouseEvent.which read-only property indicates which button was pressed on the mouse to trigger the event. The standard alternatives to this property are MouseEvent.button and MouseEvent.buttons.

Reply:
ok, just makin sure